### ISSUE TITLE
Did you train SwinIR on DIV test set?

### ISSUE THREAD

**Issue by caopulan** [JingyunLiang/SwinIR#133]
Hello, I'm reproducing the swinir results with DIV2K training set (800images) and got a lower psnr. I find that the div2k file in this repo containing its test set (900images).
